src/layouts/EditHomepage.jsx

```diff
@@ -873,6 +873,24 @@ export default class EditHomepage extends Component {
     } = this.state;
     const { match } = this.props;
     const { siteName } = match.params;
+
+    const hasSectionErrors = _.some(errors.sections, section => {
+      // Section is an object, e.g. { hero: {} }
+      // _.keys(section) produces an array with length 1
+      // The 0th element of the array contains the sectionType
+      const sectionType = _.keys(section)[0]
+      return _.some(section[sectionType], errorMessage => errorMessage.length > 0) === true
```

Comment:
(nitpick) since `_.keys(section)` produces an array with length 1, can't this be reduced to `section[sectionType].length > 0`?

Reply:
So the error object looks like this:
```
errors: {
  sections: [
    {
      hero: {
        title: '',
        subtitle: '',
        background: ''
      }
    },
    {
      infobar: {
        title: '',
        url: '//Some error message'
      }
    },
  ]
}
```

`section[sectionType].length` will return the number of fields in the section. In the given example above, both hero and infobar will return true for `section[sectionType].length > 0` even though only infobar has an error message.

Reply:
Isn't that bad tho? since hero doesn't have any errors

Reply:
Right, so instead of using

```
section[sectionType].length > 0
```

This PR uses

```

_.some(section[sectionType], errorMessage => errorMessage.length > 0) === true

```